alpenhorn/cli/options.py

```diff
@@ -463,13 +462,13 @@ def files_in_nodes(
 
             # In this mode, there's no reason to continue once the set is empty
             if not node_files:
-                return []
+                return node_files
```

Comment:
Is it better practice to return `node_files` here, or just to `break` and have a single return statement?

Reply:
Yeah, that's a good question that I don't know the answer two.  I can change it.